src/tracked_struct.rs

```diff
@@ -864,19 +874,18 @@ where
     ) -> &'db C::Fields<'db> {
         let id = AsId::as_id(&s);
         let field_ingredient_index = self.ingredient_index.successor(relative_tracked_index);
-        let data = Self::data(zalsa.table(), id);
-
-        data.read_lock(zalsa.current_revision());
+        let data = Self::data_raw(zalsa.table(), id);
 
-        let field_changed_at = data.revisions[relative_tracked_index];
+        let field_changed_at = unsafe { (&(*data).revisions)[relative_tracked_index].load() };
 
         zalsa_local.report_tracked_read_simple(
             DatabaseKeyIndex::new(field_ingredient_index, id),
-            data.durability,
+            unsafe { (*data).durability },
```

Comment:
Is it okay to acquire the lock after reporting the read and loading the revision/durability here?